src/bundle/Resources/config/services.yaml

```diff
@@ -167,3 +172,34 @@ services:
             -
               name: !php/const \Ibexa\Bundle\SystemInfo\DependencyInjection\IbexaSystemInfoExtension::METRICS_TAG
               identifier: content_types
+
+    # Services
+    Ibexa\SystemInfo\Service\ServiceProviderInterface:
+        alias: Ibexa\SystemInfo\Service\AggregateServiceProvider
+
+    Ibexa\SystemInfo\Service\AggregateServiceProvider:
+        arguments:
+            $service: !tagged_locator
+                tag: !php/const \Ibexa\Bundle\SystemInfo\DependencyInjection\IbexaSystemInfoExtension::SERVICE_TAG
+                index_by: identifier
+
+    Ibexa\SystemInfo\Service\SearchEngineService:
+        arguments:
+            $repositoryConfigProvider: '@ezpublish.api.repository_configuration_provider'
+        tags:
+            - name: !php/const \Ibexa\Bundle\SystemInfo\DependencyInjection\IbexaSystemInfoExtension::SERVICE_TAG
+              identifier: searchEngine
+
+    Ibexa\SystemInfo\Service\HttpCacheService:
+        arguments:
+            $parameterBag: '@parameter_bag'
```

Comment:
Any particular reason why you did not inject the parameter in question directly?

Reply:
I am not sure if I follow, could you please elaborate?

Reply:
You are injecting whole paremeter_bag just to extract `purge_type` from it, when you could inject only that.